gnuboardProject/backend/src/portfolio/portfolio.service.ts

```
@@ -1,14 +1,23 @@
+import { PortfolioSkill } from './entities/portfolio_skill.entity';
+import { PortfolioKeyword } from './entities/portfolio_keyword.entity';
+import { PortfolioJob } from './entities/portfolio_job.entity';
+import { PortfolioExperience } from './entities/portfolio_experience.entity';
+import { PortfolioProject } from './entities/portfolio_project.entity';
+import { PortfolioCertificate } from './entities/portfolio_certificate.entity';
+import { PortfolioLanguage } from './entities/portfolio_language.entity';
+import { PortfolioActivity } from './entities/portfolio_activity.entity';
+import { Skill } from '../skill/entities/skill.entity';
+import { Keyword } from '../keyword/entities/keyword.entity';
+import { Job } from '../job/entities/job.entity';
 import * as fs from 'fs';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)